/**
 Copyright 2022-2023 PrimeQA Team

 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at

     http://www.apache.org/licenses/LICENSE-2.0

 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
*/

syntax = "proto3";
import "google/protobuf/struct.proto";

package parameter;

message Parameter {
    string parameter_id = 1;
    string name = 2;
    string description = 3;
    string type = 4;
    google.protobuf.Value value = 5;
    repeated google.protobuf.Value options = 6;
    repeated double range = 7;
}